multiversx/mx-exchange-service
# [SERVICES-1423] fix energy outdated contracts caching

## Reasoning
- updating energy does not consistently invalidating the caches
  
## Proposed Changes
- fix farm v2 setter service
- fix multiple cache keys
- add logic to update / invalidate outdated contracts per each event

## How to test
- on xExchange after an energy update TX, no second energy update popup should appear
